### PR TITLE
fix some minor HTML incompatibilities which can cause rendering

### PR DESCRIPTION
**Pull request by mikebonnet**
inconsistencies in the browser

### CODE REVIEW

**Comment by hluk** [comment]
+1 but some tests are failing.

**Comment by mikebonnet** [comment]
@hluk This looks like a problem with CentOS CI. Not sure how to fix it.

File: datagrepper/templates/index.html [hunk 1]
```diff
@@ -44,7 +44,7 @@
   <body>
 
                 <div class="masthead">
-                  <div class="container-narrow"><img src="{{datagrepper_logo}}" height=40px /></div>
+                  <div class="container-narrow"><img src="{{datagrepper_logo}}" height="40px"></div>
```
Comment (pypingou) [inline datagrepper/templates/index.html]: Are img tags supposed to be closed with a "/" ?

**Comment by pypingou** [comment]
One comment, the rest looks good

File: datagrepper/templates/index.html (hunk 1 above)
Comment (mikebonnet) [reply datagrepper/templates/index.html]: According to https://www.w3.org/TR/2012/WD-html-markup-20120329/syntax.html#void-element `img` is a `void element` which makes the closing `/` optional. I can add it if you prefer.

Comment (pypingou) [reply datagrepper/templates/index.html]: I guess my info were outdated, thanks :)

**Pull request merged**